data/coordinator/regtest/seed

```diff
@@ -0,0 +1 @@
+hؤ�SfNv��Z���
```

Comment:
:question: Do we want to commit this seed?

Reply:
yes, otherwise we can not fix the coordinators node id for regtest.